dotcom-rendering/src/components/SignInGate/displayRule.ts

```diff
@@ -66,6 +66,16 @@ export const isValidTag = (tags: TagType[]): boolean => {
 	);
 };
 
+export const calculateArticleLimit = (
+	timeOfPageView: Date,
```

Comment:
Would making this a `number` as returned by `getHours` be simpler?

Reply:
yeah, i think it's ok as it it. It's only going likely to be in the code base for a few days